src/integrationtest/TestUtils.ts

```diff
@@ -135,6 +138,22 @@ export class TestUtils {
         await this.qldbClient.updateLedger(request).promise();
     }
 
+    async readIonValue(txn: TransactionExecutor, value: dom.Value): Promise<IonType> {
```

Comment:
Why not return the value? Instead of just the type?

Reply:
Changed to return the value itself  rather than the type.